examples/sampling/graphbolt/rgcn/hetero_rgcn.py

```diff
@@ -554,7 +554,11 @@ def train(
 
 def main(args):
     device = torch.device(
-        "cuda" if args.num_gpus > 0 and torch.cuda.is_available() else "cpu"
+        "cuda"
+        if args.num_gpus > 0
+        and torch.cuda.is_available()
+        and args.dataset != "ogb-lsc-mag240m"  # Large dataset cannot be pinned.
```

Comment:
It can be pinned on my machine, how about we make it optional to enable disk feature fetching?

Reply:
OK, I think there is no need for using a new class for feature fetcher, and I will integrate it within torch feature.

Reply:
@frozenbugs Should we decide to use disk fetcher or make it optional for user ?